src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static final Random random = new Random();
+    public static final Color[] values = Color.values();
```

Comment:
Similar to the `random` instance, the `values` array should not be `public` or `static`. It's better to make it private and non-static. Moreover, since it's a constant, you could name it `VALUES` to follow the naming convention for constants.